src/pages/AddItem/index.js

```diff
@@ -80,20 +81,29 @@ export const AddItemPage = () => {
   };
 
   return (
-    <>
-      <Header className="page-header">Add Item</Header>
-      <ContentContainer>
-        <form onSubmit={onSubmit} className="add-form">
-          <AddForm handleForm={handleForm} formState={formState} />
-          <Button type="submit" disabled={formState.productName === ''}>
-            Add Item
-          </Button>
-        </form>
-
-        {isLoading ? <p>Adding product...</p> : null}
-        {message ? <p>{message}</p> : null}
-      </ContentContainer>
-      <Navigation />
-    </>
+    <main className="page-container">
+      <div className="image-container">
+        <img src={Image} alt="Shopping" width="80%" />
+      </div>
+      <div className="card-container">
+        <ContentContainer>
+          <form onSubmit={onSubmit} className="add-form">
+            <AddForm handleForm={handleForm} formState={formState} />
```

Comment:
See, it's here. the `form` tag and then the `AddForm` component

Reply:
It had div tag, but it was a suggestion in Isa's revision to change it to form ...